WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,49 +1,38 @@
+In the last version, we introduced a donation button to gauge community interest in funding Wasabi's future. We have been overwhelmed by the support and interest shown by our community, including long-time users and privacy enthusiasts.
 
-However, maintaining and improving the software requires ongoing effort, such as maintenance, continuous improvement, bug fixes, testing, documentation, regular updates etc...
+We want to thank you for believing in Wasabi and for your support, whether through using the software, providing feedback, contributing financially, or contributing to the open-source project in any form.
 
-While Wasabi is one of the best tools available to use Bitcoin privately, achieving flawless financial-sovereign privacy is still a work in progress.
-There are many features yet to be implemented, and your support can help make them a reality.
+To show our gratitude, we've designed this release for you, our community. We've finally addressed some of the oldest bugs and feature requests that have been long-awaited!
 
-Additionally, contributors cover static costs, such as server expenses and release management, using their personal funds.
+Listing all the improvements would be too lengthy, but here are some examples:
+- Randomly skipping rounds was removed
+- Coinjoin settings & profiles are now all in a single tab
+- More information in the status icon
+- Resync button now available in Tools
+Etc...
 
-A donate button has been added on the main screen.<br/>
-It uses Silent Payments, so it's a great way to test that new feature!
+We expect these improvements to significantly enhance the overall experience of using Wasabi as your daily Bitcoin wallet. Take a tour or read release details on GitHub to see all of them!
 
-By donating, you’re directly supporting the developers who work tirelessly to maintain and improve Wasabi Wallet and make Bitcoin privacy accessible for everyone.
-
-#### 🧪 ~~TestNet 3~~ -> TestNet 4
-
-Instead of TestNet 3, Wasabi now uses TestNet 4.
-
-This new network has slightly different mechanisms that mitigates some of the problems that were undermining TestNet 3 to the point that it is basically unusable.
-
-While using the test network will be much more enjoyable now, coins from TestNet 3 don't exist on this new network. You must use a faucet and get yourself TestNet 4 coins.
-
-### 📝 Release Notes in the client
-
-If you've reached this far, you've probably noticed that the release notes highlights are now available in the client! They are displayed every first launch after an update.
-
-Full release notes are still available on GitHub.
+### 📦 Backend and Coordinator packaged for Linux
 
+The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). This update simplifies the deployment process for Linux users, making it straightforward to set up their backend servers and coordinators.
```

Comment:
```suggestion
The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). 
```
this is basically the same as next sentence?
maybe remove?

Reply:
Idk, I like what @turbolay does with the highlights, a mean for communicating views, ideas, benefits, and even to say "thank you" and not just list a condensed summary of "What's new".

Reply:
Thanks @lontivero !

@MarnixCroes I removed first line, you are right that 2nd one conveys nicely the idea already